include/exadg/operators/inverse_mass_operator.h

```diff
@@ -91,9 +109,36 @@ class InverseMassOperator
     }
   }
 
+  void
+  initialize_inverse_mass_operator_with_block_jacobi()
```

Comment:
I'm not sure this is a good name for this function. To me, this name sounds like _initialize variable with 0_, which is not what this function does, if you know what I mean.  I would suggest something like `construct_block_jacobi_preconditioner()`. What do you think?

Reply:
I agree here. For me this is a sign, that the new member function introduced here is somewhat superfluous.

Reply:
To address both reviews at once, I will remove the function and add it to `initialize`.